.pre-commit-config.yaml

```diff
@@ -0,0 +1,15 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+repos:
+  - repo: https://github.com/pre-commit/mirrors-prettier
+    rev: "v2.5.1" # Use the sha or tag you want to point at
+    hooks:
+      - id: prettier
+  # - repo: https://github.com/markdownlint/markdownlint
+  #   rev: "v0.11.0" # or specific git tag
+  #   hooks:
+  #     - id: markdownlint # Configure in .mdlrc
```

Comment:
Did you mean to comment this back in or remove?

Reply:
Good catch thanks. I was comparing two variations of markdownlint side by side, forgot to remove that one.